File: Ambush/Utils/Enums.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Ambush
{
    public class Enums
    {
        public enum State { OFF, ON }
        public enum MsgVars:int {  Action=1 , Component , VirtualId , Answer ,State,PhysicalId}
        public enum Direction { None = 0,Up,Middle,Down}

        public static string StateToString(State st)
        {
            switch(st)
            {
                case State.ON:
                    return "1";
                case State.OFF:
                    return "0";
            }
            return null;
        }
    }
}

File: Ambush/Python Scripts/enums.py
from enum import Enum
class Data(Enum):
    action = 1
    component = 2
    id = 3
    direction = 4
    physicalID = 5
class Direction(Enum):
    Up=1
    Middle = 2
    Down = 3
class Components(Enum):
    Door='DR'
    Target = 'TR'
    Sensor = 'SN'
    CPX= 'CPX'

class Action(Enum):
    Get = 'GET'
    Set = 'SET'
    Brod = 'BROD'
